client/src/components/boxes/DepositBox/DepositBoxForm/DepositBoxForm.tsx

```diff
@@ -1,32 +1,33 @@
-import { Space, Typography, Form } from 'antd';
-import s from './DepositBoxForm.module.less';
-const { Title } = Typography;
-import { DepositColumn } from '../DepositFormColumn';
-import { InputNumberBox } from '../InputNumberBox';
-import { Content } from 'antd/lib/layout/layout';
 import { FormButton } from '@/components';
-import TextArea from 'antd/lib/input/TextArea';
-import { useDepositContext } from './DepositContext';
-import { useNavigate } from 'react-router-dom';
-import { useMutation } from '@tanstack/react-query';
+import { CalculatorView } from '@/components/Calculator/CalculatorView';
 import {
-  AmountsKeys,
   APIManager,
+  AmountsKeys,
+  FormMessage,
+  MONEY_VALUES,
+  ZLOTY_AMOUNTS_KEYS,
   fetcher,
-  useAuthContext,
-  useBoxContext,
-  useSetStationUnavailableQuery,
+  isBoxExists,
+  moneyValuesType,
   recognizeError,
-  FormMessage,
+  setStationUnavailable,
+  sum,
   useAmountsQuery,
+  useBoxContext,
+  useDepositContext,
 } from '@/utils';
-import { CalculatorView } from '@/components/Calculator/View/CalculatorView';
-import { useEffect, useState } from 'react';
 import { Spinner } from '@components/Layout/Spinner/Spinner';
+import { useMutation } from '@tanstack/react-query';
+import { Form, Space, Typography } from 'antd';
+import TextArea from 'antd/lib/input/TextArea';
```

Comment:
```suggestion
import { TextArea } from 'antd/lib/input/TextArea';
```

To powinien automat robić. Nitpick